### PR TITLE
docs: update readme markdown for v2.7.2 to clarify python version requirement

### PR DESCRIPTION
**Pull request by sangshuduo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,19 @@ Or with git url:
 pip install git+https://github.com/taosdata/taos-connector-python.git
 ```
 
+Note: taospy v2.7.2 requirs Python 3.6+. The early version of taospy from v2.5.0 to v2.7.1 requires Python 3.7+.
```
Comment (zitsen) [inline README.md:23]: earlier versions... require...

File: README.md [hunk 2]
```diff
@@ -20,11 +20,19 @@ Or with git url:
...
+
 ## Source Code
 
 [TDengine](https://github.com/taosdata/TDengine) connector for Python source code is hosted on [GitHub](https://github.com/taosdata/taos-connector-python).
 
-## Examples for `taosws` Module
+## Install taos-ws-py
+
+```bash
+pip install taos-ws-py
```
Comment (zitsen) [inline README.md:32]: pip install "taospy[ws]" will install taos-ws-py along with taospy.

**Review by zitsen — COMMENTED** [review]
LGTM with some comments

**Review by zitsen — APPROVED** [review]

**Review by sunpe — APPROVED** [review]
LGTM

**Pull request merged**